### PR TITLE
chore: apk requirements to run mender-artifact on alpine

### PR DESCRIPTION
**Pull request by 0lmi**
Changelog: None
Ticket: MEN-5906
Signed-off-by: Alex Miliukov <oleksandr.miliukov@northern.tech>

### CODE REVIEW

File: apk-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+libc6-compat
+xz-dev
```
Comment (lluiscampos) [inline apk-requirements.txt:2]: We need `xz` instead of `xz-dev`, right? These are supposed to be runtime requirements?

Comment (lluiscampos) [inline apk-requirements.txt:2]: We are also missing libssl. It is installed by default in Alpine Linux, but I think is good to be explicit (specially if they change to libssl3 any time soon).
```suggestion
xz-dev
libssl1.1
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: apk-requirements.txt (hunk 1 above)
Comment (0lmi) [reply apk-requirements.txt:2]: I don't really know those details. Let me check...

Comment (0lmi) [reply apk-requirements.txt]: Yep, you are totally right 👍

Comment (0lmi) [reply apk-requirements.txt]: 👍

**Review by lluiscampos — APPROVED** [review]

**Comment by 0lmi** [comment]
@mender-test-bot start pipeline

**Pull request merged**